scripts/release/mule/test/test.sh

```diff
@@ -16,9 +16,15 @@ export CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
 export NETWORK=${NETWORK:-$(./scripts/compute_branch_network.sh "$BRANCH")}
 export SHA=${SHA:-$(git rev-parse HEAD)}
 export VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
+ALGORAND_PACKAGE_NAME=$([ "$CHANNEL" = beta ] && echo algorand-beta || echo algorand)
+DEVTOOLS_PACKAGE_NAME=$([ "$CHANNEL" = beta ] && echo algorand-devtools-beta || echo algorand-devtools)
+export ALGORAND_PACKAGE_NAME
+export DEVTOOLS_PACKAGE_NAME
 
+# Note that currently we are only supporting amd64 packages (20201030).
```

Comment:
Do we need this restriction? I think we're packaging all architectures now, so it's feasible to test all.

Reply:
Yes, that's true, I was thinking that we only support amd64 from our repos.  I'll remove this.